<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroalterar</class>
 <widget class="QDialog" name="cadastroalterar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alterar Dados</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF</string>
  </property>
  <widget class="QWidget" name="sidebar" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9B88BF; border-radius: 20px</string>
   </property>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C4B4E0;
font: 700 9pt &quot;System&quot;;
border-radius: 12px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="consultaButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C4B4E0;
font: 700 9pt &quot;System&quot;;
border-radius: 12px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>Consulta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agendamentoButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C4B4E0;
font: 700 9pt &quot;System&quot;;
border-radius: 12px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>Agendamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-100</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/resources.qrc">:/images/images/logo4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>agendamentoButton</zorder>
   <zorder>label</zorder>
   <zorder>menuButton</zorder>
   <zorder>consultaButton</zorder>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>971</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9B88BF; border-radius: 15px</string>
   </property>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Garamond&quot;; color: #ffffff</string>
    </property>
    <property name="text">
     <string>Alterar Dados Clientes</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="data_background" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>971</width>
     <height>461</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #C7C0D5; border-radius: 20px</string>
   </property>
   <widget class="QPushButton" name="cadastrarButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9B88BF;
font: 700 15pt &quot;System&quot;;
border-radius: 15px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelarButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #EE6F6F;
font: 700 15pt &quot;System&quot;;
border-radius: 15px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>22</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="nomeTutor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">

QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}

; background-color: #ffffff</string>
       </property>
       <property name="placeholderText">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cpfTutor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">

QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}

; background-color: #ffffff</string>
       </property>
       <property name="placeholderText">
        <string>CPF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="emailTutor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">

QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}

; background-color: #ffffff</string>
       </property>
       <property name="placeholderText">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="enderecoTutor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">

QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}

; background-color: #ffffff</string>
       </property>
       <property name="placeholderText">
        <string>Endere√ßo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="telefoneTutor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">

QLineEdit {
    font-size: 14px;
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 5px;
}
QLineEdit:placeholder {
    color: gray;
    font-style: italic;
}

; background-color: #ffffff</string>
       </property>
       <property name="placeholderText">
        <string>Telefone</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="alterarButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9B88BF;
font: 700 15pt &quot;System&quot;;
border-radius: 15px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>ALTERAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="listarButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9B88BF;
font: 700 15pt &quot;System&quot;;
border-radius: 15px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>LISTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apagarButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9B88BF;
font: 700 15pt &quot;System&quot;;
border-radius: 15px;
padding: 5px;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>APAGAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
